com.unity.render-pipelines.high-definition/Runtime/Lighting/ProbeVolume/ProbeVolume.cs

```diff
@@ -263,11 +266,11 @@ public static void SetSphericalHarmonicsL1FromIndex(ref ProbeVolumePayload paylo
             payload.dataSHL01[indexDataBaseSHL01 + 3] = sh.shAr.x;
             payload.dataSHL01[indexDataBaseSHL01 + 4] = sh.shAr.y;
             payload.dataSHL01[indexDataBaseSHL01 + 5] = sh.shAr.z;
-            
+
             payload.dataSHL01[indexDataBaseSHL01 + 6] = sh.shAg.x;
             payload.dataSHL01[indexDataBaseSHL01 + 7] = sh.shAg.y;
             payload.dataSHL01[indexDataBaseSHL01 + 8] = sh.shAg.z;
-            
```

Comment:
What's with all the whitespace changes?

Reply:
usually it is the auto format tool that remove unnecessary white space. It happen because in a previous PR, someone have certainly not enable it :)